test/e2e/framework/node/resource.go

```diff
@@ -563,18 +563,14 @@ func GetClusterZones(c clientset.Interface) (sets.String, error) {
 
 // GetSchedulableClusterZones returns the values of zone label collected from all nodes which are schedulable.
 func GetSchedulableClusterZones(c clientset.Interface) (sets.String, error) {
-	nodes, err := c.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
+	nodes, err := GetReadySchedulableNodes(c)
```

Comment:
should we add a comment that - now this function skips zones which have tainted nodes?

Reply:
A comment added